src/pkg/message/message.go

```diff
@@ -243,6 +244,17 @@ func Paragraphn(n int, format string, a ...any) string {
 	return pterm.DefaultParagraph.WithMaxWidth(n).Sprintf(format, a...)
 }
 
+// PrintDiff prints the differences between a and b with a as original and b as new
+func PrintDiff(textA, textB string) {
```

Comment:
Does `diffmatchpatch` colorize the output like Git diff does?

![Screenshot 2023-04-25 at 5 54 46 PM](https://user-images.githubusercontent.com/50058333/234424654-282f5c69-f65a-4276-9ba2-71a9842c25fa.png)

Reply:
It is not quite the same but yes it does - it does it based on character not line:
![image](https://user-images.githubusercontent.com/3977569/234428220-713d87a4-f330-4a27-b3f9-556bb9d0155d.png)